deployments/kubernetes/install/helm/istio-cni/values.yaml

```diff
@@ -1,5 +1,5 @@
 hub: gcr.io/istio-release
-tag: master-latest-daily
+tag: release-1.2-latest-daily
```

Comment:
@tiswanso this is not a gke override -this is the main config.  The test-infra seds (iirc) master-latest-daily to produce the version number.

Reply:
yeah, my bad... there was a similar fix yesterday on master for the gke file only and I assumed this was a cherrypick.